### PR TITLE
Web Storage

### PR DESCRIPTION
**Pull request by mjzac**

### CODE REVIEW

File: web-storage/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Text
+
+Web Storage provides mechanisms to store key value pairs in a much more intuitive way than using cookies.
+
+There are two different APIs available:
+
+- sessionStorage
+
+  - The values are stored only for the duration of page session. The values get cleared when the tab is closed.
```
Comment (avishekjana) [inline web-storage/README.md:9]: "The values are stored only for the duration of page session"
- Can be explained in more detail in one or two sentences. Like: what is page session?

File: web-storage/README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Text
+
+Web Storage provides mechanisms to store key value pairs in a much more intuitive way than using cookies.
```
Comment (avishekjana) [inline web-storage/README.md:3]: Please add one or two more sentences here about web storage and why it's different from cookies.

File: web-storage/README.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  // Remove saved data from sessionStorage
+  sessionStorage.removeItem("name");
+
+  // Remove all saved data from sessionStorage
+  sessionStorage.clear();
+  ```
+
+- localStorage
+
+  - localStorage is similar to sessionStorage, but it doesn't have an expiry.
```
Comment (avishekjana) [inline web-storage/README.md:31]: Also add few more points here, like:

- The keys and the values are always strings
- Data stored in localStorage, is never transferred to the server.
- Data is shared between all tabs and windows from the same origin.

File: web-storage/README.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+  // Get saved data from localStorage
+  const themePreference = localStorage.getItem("themePreference");
+
+  // Remove saved data from localStorage
+  localStorage.removeItem("themePreference");
+
+  // Remove all saved data from localStorage
+  localStorage.clear();
+  ```
```
Comment (avishekjana) [inline web-storage/README.md:47]: Also show, "How to save js Objects in localStorage?".

If we need to save an Object in localStorage, we need to stringify it. That is, using JSON#stringify API we need to convert the JavaScript value to a JavaScript Object Notation JSON string.

File: web-storage/README.md (hunk 1 above)
Comment (mjzac) [reply web-storage/README.md]: @avishekjana I have updated the content

**Pull request merged**